grunt/config/watch.js

```diff
@@ -18,11 +18,11 @@ module.exports = {
     },
     courseJson: {
         files: ['<%= sourcedir %>course/**/*.<%= jsonext %>'],
-        tasks : ['jsonlint', 'check-json', 'copy:courseJson', 'schema-defaults', 'create-json-config']
+        tasks : ['jsonlint', 'check-json', 'newer:copy:courseJson', 'schema-defaults', 'create-json-config']
```

Comment:
did the inclusion of the `newer:` library give any benefits?

Reply:
Yes, it facilitates to copy only modified files instead of all the files.

Reply:
is it faster?

Reply:
yes, as compared to the previous one.

Reply:
it looks as though newer only compares file modified times one way 
https://github.com/tschaub/grunt-newer/blob/master/lib/util.js#L27-L28

this means it probably won't work if i paste an older file over a newer file?

Reply:
If the file in build folder the same as the file in src folder then it doesn't copy to build folder.when you paste an older file over a newer file. watch task listens to the changes and calls newer:copy task and it works in the above case also.